knative-operator/pkg/apis/operator/v1alpha1/knativekafka_types.go

```diff
@@ -54,6 +57,17 @@ type KnativeKafkaList struct {
 	Items           []KnativeKafka `json:"items"`
 }
 
+// Broker allows configuration for KafkaBroker installation
+type Broker struct {
+	// Enabled defines if the KafkaBroker installation is enabled
+	Enabled bool `json:"enabled"`
+
+	// BootstrapServers is comma separated string of bootstrapservers that the
+	// brokers will use
```

Comment:
Can we clarify that this is the default, but it doesn't have to be the same for all Kafka Brokers?
Folks can still specify a different list if they want.
```suggestion
	// BootstrapServers is the default comma-separated string of bootstrapservers that the
	// brokers will use
```

Reply:
good point! Will change the text

Reply:
> 	// BootstrapServers is the default comma-separated string of bootstrapservers that the
	// brokers will use, but can be on the individual broker object's config map.